ccsdspy/packet_fields.py

```diff
@@ -9,11 +9,15 @@
 
 import numpy as np
 
+from .converters import Converter
+
 
 class PacketField:
     """A field contained in a packet."""
 
-    def __init__(self, name, data_type, bit_length, bit_offset=None, byte_order="big"):
+    def __init__(
+        self, name, data_type, bit_length, bit_offset=None, byte_order="big", converter=None
```

Comment:
If converters are added after object creation, you don't need the `converter` keyword here anymore.

Reply:
You're right; this is something I missed